source/language/openpulse.rst

```diff
@@ -30,227 +31,249 @@ could even reference multiple qubits in a chain.
 
 Channels are vendor-specific and device-specific. It is expected that vendors
 of quantum hardware provide the appropriate channel names and qubit mappings
-as configuration information to end users. Each channel may also have associated 
-static settings, such as local-oscillator frequencies, which do not vary 
-throughout program execution. Again it is expected that vendors of quantum 
+as configuration information to end users. Each channel may also have associated
+static settings, such as local-oscillator frequencies, which do not vary
+throughout program execution. Again it is expected that vendors of quantum
 hardware provide a method for manipulating those static settings if appropriate.
 
 There are two kinds of channels: transmit channels (sending input to a quantum
 device) and receive channels (reading output from a quantum device).
```

Comment:
We should include the syntax for channels, in particular the channel accessor syntax.